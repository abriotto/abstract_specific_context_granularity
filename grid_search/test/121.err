  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:13,  7.06it/s]  3%|#####6                                                                                                                                                                                      | 3/99 [00:00<00:10,  9.43it/s]  5%|#########4                                                                                                                                                                                  | 5/99 [00:00<00:09,  9.88it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:07, 12.49it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:07, 11.28it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:00<00:06, 12.75it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:07, 11.91it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:06, 11.89it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:01<00:07, 10.32it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:01<00:06, 11.49it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:02<00:07, 10.44it/s] 24%|#############################################3                                                                                                                                             | 24/99 [00:02<00:07,  9.77it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:08,  8.66it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:02<00:08,  8.78it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:02<00:07,  9.53it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:07,  8.97it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:06,  9.84it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:03<00:07,  8.35it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:03<00:07,  8.23it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:06,  9.82it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:06,  9.38it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:05,  9.61it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:04<00:05,  9.16it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:04<00:05,  9.35it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:06,  8.27it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:04, 10.12it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:04,  9.30it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:05<00:04,  8.84it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:05<00:05,  8.25it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:05,  7.48it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:04,  8.98it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:04,  8.33it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:06<00:03, 10.13it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:06<00:03,  9.43it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:07<00:03,  8.86it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:03,  8.17it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:07<00:03,  9.56it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:07<00:03,  8.04it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:07<00:02,  9.55it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:02,  8.24it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  7.67it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:08<00:02,  7.19it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:08<00:02,  8.58it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:08<00:02,  8.08it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:01,  8.05it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:01,  7.63it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:09<00:01,  7.10it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01,  6.76it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:09<00:01,  9.16it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:09<00:01,  8.86it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  8.06it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  7.59it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  7.20it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:10<00:00,  6.67it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  6.59it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:10<00:00,  6.45it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00,  6.80it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.82it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:03,  6.07it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:02,  8.79it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:02,  7.76it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:02,  6.96it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:02,  7.31it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:01<00:02,  6.97it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:01<00:02,  6.56it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:01<00:02,  6.33it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01,  7.12it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  7.46it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:02<00:00,  9.45it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:02<00:00,  7.93it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  9.26it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00, 10.96it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  9.87it/s]100%|###########################################################################################################################################################################################| 25/25 [00:03<00:00,  8.22it/s]
