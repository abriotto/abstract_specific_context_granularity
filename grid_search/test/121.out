# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6960753202438354, "acc": 0.5086778402328491, "length": 3.5701119899749756, "mode": "train", "epoch": 1}
{"loss": 0.6918184161186218, "acc": 0.5092810988426208, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6916332840919495, "acc": 0.5241066813468933, "length": 3.4854352474212646, "mode": "train", "epoch": 2}
{"loss": 0.6919137835502625, "acc": 0.5232546925544739, "length": 1.0016722679138184, "mode": "test", "epoch": 2}
{"loss": 0.687516987323761, "acc": 0.5474339723587036, "length": 3.285907030105591, "mode": "train", "epoch": 3}
{"loss": 0.6749991774559021, "acc": 0.6249470710754395, "length": 1.9832775592803955, "mode": "test", "epoch": 3}
{"loss": 0.673392117023468, "acc": 0.6104205846786499, "length": 3.1684818267822266, "mode": "train", "epoch": 4}
{"loss": 0.6470776200294495, "acc": 0.6760767102241516, "length": 3.9247491359710693, "mode": "test", "epoch": 4}
{"loss": 0.6349174976348877, "acc": 0.6665412187576294, "length": 3.7295422554016113, "mode": "train", "epoch": 5}
{"loss": 0.5963910222053528, "acc": 0.6902174353599548, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5889489650726318, "acc": 0.6850932240486145, "length": 3.9517982006073, "mode": "train", "epoch": 6}
{"loss": 0.5937721133232117, "acc": 0.6924750208854675, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5750407576560974, "acc": 0.698191225528717, "length": 3.970304012298584, "mode": "train", "epoch": 7}
{"loss": 0.5786066055297852, "acc": 0.7159698605537415, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5636265277862549, "acc": 0.7111848592758179, "length": 3.978325366973877, "mode": "train", "epoch": 8}
{"loss": 0.5605661273002625, "acc": 0.7211537957191467, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.548140823841095, "acc": 0.7187531590461731, "length": 3.9836323261260986, "mode": "train", "epoch": 9}
{"loss": 0.5375192761421204, "acc": 0.7219899892807007, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5336953401565552, "acc": 0.7259652018547058, "length": 3.988351583480835, "mode": "train", "epoch": 10}
{"loss": 0.5261150598526001, "acc": 0.7375417947769165, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5218701958656311, "acc": 0.7420741319656372, "length": 3.990872621536255, "mode": "train", "epoch": 11}
{"loss": 0.5082179307937622, "acc": 0.7607022523880005, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.510233998298645, "acc": 0.7544352412223816, "length": 3.995342254638672, "mode": "train", "epoch": 12}
{"loss": 0.5010414719581604, "acc": 0.7626252770423889, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.49785786867141724, "acc": 0.7611860632896423, "length": 3.995190143585205, "mode": "train", "epoch": 13}
{"loss": 0.4935629963874817, "acc": 0.7650502920150757, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.48729830980300903, "acc": 0.7695764899253845, "length": 3.9978256225585938, "mode": "train", "epoch": 14}
{"loss": 0.47907599806785583, "acc": 0.7710703015327454, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.4749070107936859, "acc": 0.7764638662338257, "length": 3.996563196182251, "mode": "train", "epoch": 15}
{"loss": 0.4658794403076172, "acc": 0.7839465141296387, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4663122296333313, "acc": 0.784820556640625, "length": 3.9986069202423096, "mode": "train", "epoch": 16}
{"loss": 0.4622257351875305, "acc": 0.7825251221656799, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.45767155289649963, "acc": 0.7896838784217834, "length": 3.998931884765625, "mode": "train", "epoch": 17}
{"loss": 0.4558298587799072, "acc": 0.7887125015258789, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.45268145203590393, "acc": 0.7911220192909241, "length": 3.9982833862304688, "mode": "train", "epoch": 18}
{"loss": 0.4496398866176605, "acc": 0.8017557859420776, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.44561654329299927, "acc": 0.7980079650878906, "length": 3.999311923980713, "mode": "train", "epoch": 19}
{"loss": 0.44536295533180237, "acc": 0.8016721606254578, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.44069981575012207, "acc": 0.8002467751502991, "length": 3.9986400604248047, "mode": "train", "epoch": 20}
{"loss": 0.4430534541606903, "acc": 0.8040969371795654, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4365431070327759, "acc": 0.8037448525428772, "length": 3.9983503818511963, "mode": "train", "epoch": 21}
{"loss": 0.4440424144268036, "acc": 0.7968225479125977, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.42938995361328125, "acc": 0.8056135773658752, "length": 3.9974725246429443, "mode": "train", "epoch": 22}
{"loss": 0.43883076310157776, "acc": 0.8069397211074829, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4242202639579773, "acc": 0.8105092644691467, "length": 3.9981465339660645, "mode": "train", "epoch": 23}
{"loss": 0.43097424507141113, "acc": 0.803260862827301, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.41439396142959595, "acc": 0.81371009349823, "length": 3.9997658729553223, "mode": "train", "epoch": 24}
{"loss": 0.4221876561641693, "acc": 0.8080268502235413, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.40319958329200745, "acc": 0.8178457021713257, "length": 3.9991164207458496, "mode": "train", "epoch": 25}
{"loss": 0.4054604470729828, "acc": 0.8142141699790955, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3888307511806488, "acc": 0.83088618516922, "length": 3.999500274658203, "mode": "train", "epoch": 26}
{"loss": 0.40056905150413513, "acc": 0.8227425813674927, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.37918609380722046, "acc": 0.8327752351760864, "length": 3.9998843669891357, "mode": "train", "epoch": 27}
{"loss": 0.39030155539512634, "acc": 0.8288460969924927, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3733151853084564, "acc": 0.8390006422996521, "length": 3.999922513961792, "mode": "train", "epoch": 28}
{"loss": 0.3830694854259491, "acc": 0.8320232629776001, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3670300245285034, "acc": 0.8387411832809448, "length": 3.9995505809783936, "mode": "train", "epoch": 29}
{"loss": 0.37782785296440125, "acc": 0.8310199975967407, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3637026250362396, "acc": 0.8400045037269592, "length": 3.9998879432678223, "mode": "train", "epoch": 30}
{"loss": 0.3730342388153076, "acc": 0.8368728160858154, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.35913145542144775, "acc": 0.8448352813720703, "length": 3.9999630451202393, "mode": "train", "epoch": 31}
{"loss": 0.3641279637813568, "acc": 0.8421406149864197, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3560231029987335, "acc": 0.848020076751709, "length": 3.999420642852783, "mode": "train", "epoch": 32}
{"loss": 0.3659075200557709, "acc": 0.8424746990203857, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.34814324975013733, "acc": 0.8528454899787903, "length": 3.999994993209839, "mode": "train", "epoch": 33}
{"loss": 0.35975462198257446, "acc": 0.8435620069503784, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3447299897670746, "acc": 0.8507251143455505, "length": 3.9999961853027344, "mode": "train", "epoch": 34}
{"loss": 0.3576444685459137, "acc": 0.8459866642951965, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3369707465171814, "acc": 0.8550154566764832, "length": 3.9999141693115234, "mode": "train", "epoch": 35}
{"loss": 0.35381779074668884, "acc": 0.8515886068344116, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3331209123134613, "acc": 0.8599376678466797, "length": 3.9997334480285645, "mode": "train", "epoch": 36}
{"loss": 0.3504016101360321, "acc": 0.852424681186676, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3268260061740875, "acc": 0.8618857264518738, "length": 3.999995708465576, "mode": "train", "epoch": 37}
{"loss": 0.341360479593277, "acc": 0.8505016565322876, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3253515362739563, "acc": 0.8598490357398987, "length": 3.99999737739563, "mode": "train", "epoch": 38}
{"loss": 0.3376500606536865, "acc": 0.8525081276893616, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.32236751914024353, "acc": 0.8652063608169556, "length": 3.9999992847442627, "mode": "train", "epoch": 39}
{"loss": 0.3361244797706604, "acc": 0.8608695268630981, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3206251859664917, "acc": 0.8666290640830994, "length": 3.999994993209839, "mode": "train", "epoch": 40}
{"loss": 0.33457431197166443, "acc": 0.8597826361656189, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.31572413444519043, "acc": 0.8673548102378845, "length": 3.9999985694885254, "mode": "train", "epoch": 41}
{"loss": 0.336899071931839, "acc": 0.856270968914032, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.31996551156044006, "acc": 0.8657084703445435, "length": 3.9999985694885254, "mode": "train", "epoch": 42}
{"loss": 0.3381921350955963, "acc": 0.8616220951080322, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3167792856693268, "acc": 0.8664060831069946, "length": 3.9999988079071045, "mode": "train", "epoch": 43}
{"loss": 0.33682820200920105, "acc": 0.861789345741272, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.31530657410621643, "acc": 0.8676060438156128, "length": 3.999999761581421, "mode": "train", "epoch": 44}
{"loss": 0.3336420953273773, "acc": 0.8635451793670654, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3129415512084961, "acc": 0.8681080937385559, "length": 3.999997854232788, "mode": "train", "epoch": 45}
{"loss": 0.32425662875175476, "acc": 0.8659699559211731, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3123762607574463, "acc": 0.8708664774894714, "length": 3.9999542236328125, "mode": "train", "epoch": 46}
{"loss": 0.33412033319473267, "acc": 0.86371248960495, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3128163814544678, "acc": 0.8682447075843811, "length": 3.999699831008911, "mode": "train", "epoch": 47}
{"loss": 0.33662524819374084, "acc": 0.8632941246032715, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3084230422973633, "acc": 0.8738280534744263, "length": 3.999999523162842, "mode": "train", "epoch": 48}
{"loss": 0.33118462562561035, "acc": 0.8640467524528503, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3070407807826996, "acc": 0.8721537590026855, "length": 3.99999737739563, "mode": "train", "epoch": 49}
{"loss": 0.3294547498226166, "acc": 0.8677257895469666, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.30624160170555115, "acc": 0.8708704710006714, "length": 3.999999761581421, "mode": "train", "epoch": 50}
{"loss": 0.3240397870540619, "acc": 0.8658025860786438, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.30468448996543884, "acc": 0.8722934722900391, "length": 3.9999983310699463, "mode": "train", "epoch": 51}
{"loss": 0.32312846183776855, "acc": 0.8663879036903381, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.30220308899879456, "acc": 0.8742186427116394, "length": 3.9999988079071045, "mode": "train", "epoch": 52}
{"loss": 0.32466021180152893, "acc": 0.8696489334106445, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.30044642090797424, "acc": 0.8768133521080017, "length": 3.99999737739563, "mode": "train", "epoch": 53}
{"loss": 0.3193398118019104, "acc": 0.8699832558631897, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.29802095890045166, "acc": 0.8780131936073303, "length": 3.999998092651367, "mode": "train", "epoch": 54}
{"loss": 0.32622310519218445, "acc": 0.8690637946128845, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2982090711593628, "acc": 0.8767019510269165, "length": 3.999999523162842, "mode": "train", "epoch": 55}
{"loss": 0.3224565386772156, "acc": 0.8693979382514954, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.29857856035232544, "acc": 0.8768693804740906, "length": 3.9999992847442627, "mode": "train", "epoch": 56}
{"loss": 0.3174395263195038, "acc": 0.8705686330795288, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.29729899764060974, "acc": 0.8777334094047546, "length": 3.999990224838257, "mode": "train", "epoch": 57}
{"loss": 0.31933802366256714, "acc": 0.8709030151367188, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2966092526912689, "acc": 0.8765903115272522, "length": 4.0, "mode": "train", "epoch": 58}
{"loss": 0.31871137022972107, "acc": 0.8704851269721985, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.2975448668003082, "acc": 0.8769810795783997, "length": 4.0, "mode": "train", "epoch": 59}
{"loss": 0.3311621844768524, "acc": 0.8647156953811646, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3062351942062378, "acc": 0.8741629719734192, "length": 3.999999523162842, "mode": "train", "epoch": 60}
{"loss": 0.32759425044059753, "acc": 0.8678094744682312, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.29594606161117554, "acc": 0.878180205821991, "length": 3.999993085861206, "mode": "train", "epoch": 61}
{"loss": 0.3315180242061615, "acc": 0.8688961863517761, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2946031093597412, "acc": 0.8795758485794067, "length": 4.0, "mode": "train", "epoch": 62}
{"loss": 0.3272249400615692, "acc": 0.8717390894889832, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.2931293547153473, "acc": 0.8788504004478455, "length": 3.999999761581421, "mode": "train", "epoch": 63}
{"loss": 0.3190317153930664, "acc": 0.8712376356124878, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.29251325130462646, "acc": 0.878766655921936, "length": 3.999999761581421, "mode": "train", "epoch": 64}
{"loss": 0.3177616000175476, "acc": 0.8688127994537354, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.2934567332267761, "acc": 0.8784318566322327, "length": 3.9999992847442627, "mode": "train", "epoch": 65}
{"loss": 0.32704272866249084, "acc": 0.8642138242721558, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.2944709360599518, "acc": 0.8772878646850586, "length": 4.0, "mode": "train", "epoch": 66}
{"loss": 0.32745954394340515, "acc": 0.8698996901512146, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.2909516990184784, "acc": 0.8795477747917175, "length": 3.999999523162842, "mode": "train", "epoch": 67}
{"loss": 0.31768444180488586, "acc": 0.8724914193153381, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.2901059091091156, "acc": 0.8797154426574707, "length": 4.0, "mode": "train", "epoch": 68}
{"loss": 0.32616671919822693, "acc": 0.8710700869560242, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.2907775938510895, "acc": 0.8799664378166199, "length": 4.0, "mode": "train", "epoch": 69}
{"loss": 0.32637712359428406, "acc": 0.8724080324172974, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2911688983440399, "acc": 0.8788224458694458, "length": 4.0, "mode": "train", "epoch": 70}
{"loss": 0.3153710961341858, "acc": 0.86981600522995, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.2914184033870697, "acc": 0.8790735006332397, "length": 3.9999985694885254, "mode": "train", "epoch": 71}
{"loss": 0.3205128610134125, "acc": 0.8704850077629089, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.28937971591949463, "acc": 0.8790178298950195, "length": 4.0, "mode": "train", "epoch": 72}
{"loss": 0.3191711902618408, "acc": 0.8712372779846191, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.2888016700744629, "acc": 0.8798269629478455, "length": 4.0, "mode": "train", "epoch": 73}
{"loss": 0.3227587342262268, "acc": 0.8708195090293884, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.28818485140800476, "acc": 0.8801059722900391, "length": 4.0, "mode": "train", "epoch": 74}
{"loss": 0.32291412353515625, "acc": 0.8707358241081238, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.2898377776145935, "acc": 0.8798548579216003, "length": 3.9999988079071045, "mode": "train", "epoch": 75}
{"loss": 0.3172365128993988, "acc": 0.8708192706108093, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.2872023582458496, "acc": 0.8818359375, "length": 4.0, "mode": "train", "epoch": 76}
{"loss": 0.3123147487640381, "acc": 0.8719899654388428, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.2864499092102051, "acc": 0.8797990083694458, "length": 3.999999761581421, "mode": "train", "epoch": 77}
{"loss": 0.3182770907878876, "acc": 0.8705683946609497, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.2854264974594116, "acc": 0.8800501823425293, "length": 3.999999761581421, "mode": "train", "epoch": 78}
{"loss": 0.31546807289123535, "acc": 0.8696489334106445, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.2855796217918396, "acc": 0.8806639313697815, "length": 4.0, "mode": "train", "epoch": 79}
{"loss": 0.3152385652065277, "acc": 0.8726588487625122, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.28517717123031616, "acc": 0.8815008401870728, "length": 3.9999969005584717, "mode": "train", "epoch": 80}
{"loss": 0.3139120042324066, "acc": 0.871404767036438, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.2864665687084198, "acc": 0.8804408311843872, "length": 4.0, "mode": "train", "epoch": 81}
{"loss": 0.3224857747554779, "acc": 0.8689798712730408, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.28789031505584717, "acc": 0.8797896504402161, "length": 3.999908924102783, "mode": "train", "epoch": 82}
{"loss": 0.3177369236946106, "acc": 0.8727425336837769, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.28686830401420593, "acc": 0.8799663782119751, "length": 3.999999761581421, "mode": "train", "epoch": 83}
{"loss": 0.3107750415802002, "acc": 0.8725752234458923, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.2841132879257202, "acc": 0.8812214732170105, "length": 3.9999942779541016, "mode": "train", "epoch": 84}
{"loss": 0.31482407450675964, "acc": 0.8726588487625122, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.28672441840171814, "acc": 0.8796535134315491, "length": 3.999312162399292, "mode": "train", "epoch": 85}
{"loss": 0.31779417395591736, "acc": 0.8704847097396851, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.28668463230133057, "acc": 0.8795760273933411, "length": 3.999999523162842, "mode": "train", "epoch": 86}
{"loss": 0.3206867277622223, "acc": 0.8680601716041565, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.28586825728416443, "acc": 0.8797988891601562, "length": 3.9999992847442627, "mode": "train", "epoch": 87}
{"loss": 0.31331273913383484, "acc": 0.8706521987915039, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.2828138470649719, "acc": 0.8816964030265808, "length": 4.0, "mode": "train", "epoch": 88}
{"loss": 0.31577345728874207, "acc": 0.8727424144744873, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.28336888551712036, "acc": 0.8802174925804138, "length": 3.9999988079071045, "mode": "train", "epoch": 89}
{"loss": 0.3101331889629364, "acc": 0.8732441663742065, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.28158289194107056, "acc": 0.8814730644226074, "length": 3.9999992847442627, "mode": "train", "epoch": 90}
{"loss": 0.3100907802581787, "acc": 0.8714883327484131, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.28216707706451416, "acc": 0.88194739818573, "length": 3.9999983310699463, "mode": "train", "epoch": 91}
{"loss": 0.309510201215744, "acc": 0.8733276724815369, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.28208744525909424, "acc": 0.8819754719734192, "length": 4.0, "mode": "train", "epoch": 92}
{"loss": 0.31050756573677063, "acc": 0.8729096055030823, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.28079670667648315, "acc": 0.8822823166847229, "length": 4.0, "mode": "train", "epoch": 93}
{"loss": 0.31644657254219055, "acc": 0.8712374567985535, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.2822452187538147, "acc": 0.8809989094734192, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.31988048553466797, "acc": 0.8698996901512146, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.2828487157821655, "acc": 0.8814452886581421, "length": 4.0, "mode": "train", "epoch": 95}
{"loss": 0.3230319619178772, "acc": 0.8725753426551819, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.28304487466812134, "acc": 0.8803851008415222, "length": 4.0, "mode": "train", "epoch": 96}
{"loss": 0.3209380805492401, "acc": 0.8737459182739258, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.28373605012893677, "acc": 0.8811104893684387, "length": 4.0, "mode": "train", "epoch": 97}
{"loss": 0.3224693834781647, "acc": 0.8707358241081238, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.28296175599098206, "acc": 0.8815569281578064, "length": 4.0, "mode": "train", "epoch": 98}
{"loss": 0.3193318545818329, "acc": 0.8708195090293884, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.2828226387500763, "acc": 0.8820032477378845, "length": 4.0, "mode": "train", "epoch": 99}
{"loss": 0.3210600018501282, "acc": 0.8705687522888184, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.28046518564224243, "acc": 0.8817798495292664, "length": 3.9999961853027344, "mode": "train", "epoch": 100}
{"loss": 0.3187181055545807, "acc": 0.873411238193512, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.8284426927566528
